modules/lotamePanoramaIdSystem.js

```diff
@@ -68,10 +76,10 @@ function setProfileId(profileId) {
  */
 function getProfileId() {
   let profileId;
-  if (storage.cookiesAreEnabled()) {
+  if (cookiesAreEnabled(false)) {
```

Comment:
looks like a couple places uses of this function that could benefit from DO_NOT_HONOR_CONFIG as well